tox.ini

```diff
@@ -46,6 +46,7 @@ description = Run unit tests
 deps =
     pytest
     coverage[toml]
+    dbus-fast>=1.90.2
```

Comment:
Shouldn't this go in the requirements.txt file, which will also be used for charm packing?